packages/vsce/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ All notable changes to the "cics-extension-for-zowe" extension will be documente
 
 ## `3.12.0`
 
+- Enhancement: Added JVM Endpoint to the JVM Server node. [#413](https://github.com/zowe/cics-for-zowe-client/issues/413)
```

Comment:
This needs to be under Recent Changes, like your other changelog. It was not part of the 3.12 release.

Also should this be the same issue as above (410)?

Reply:
This is not yet resolved - it should be pointing to the issue rather than the PR